xen-bugtool

```diff
@@ -458,7 +458,7 @@ cap(CAP_PERSISTENT_STATS,    PII_NO,                    max_size=50*MB,
 cap(CAP_PROCESS_LIST,        PII_YES,                   max_size=30*KB,
     max_time=60)
 cap(CAP_SYSTEM_LOAD,         PII_MAYBE,                 max_size=70*MB, max_time=30)
-cap(CAP_SYSTEM_LOGS,         PII_MAYBE,                 max_size=50*MB,
+cap(CAP_SYSTEM_LOGS,         PII_MAYBE,                 max_size=500*MB,
```

Comment:
AFAIK `CAP_SYSTEM_LOGS` is controlled by the verbosity flag which chooses how many days' worth of logs to include and therefore the size should be ignored. However, it appears to be missing a call to `update_cap_size()` to set the size dynamically.

See the similar call for `CAP_XENSERVER_LOGS` around line 1292.